packages/react-app/src/components/bridge/ToToken.jsx

```diff
@@ -154,6 +188,19 @@ export const ToToken = () => {
                 {utils.formatUnits(amount, token.decimals)}
               </Text>
             )}
+            {bridgeDirection !== ETH_XDAI_BRIDGE && // Temporary block and will be removed once the audit is finished.
```

Comment:
Should disable from home to foreign and foreign to home for now until audit is finished.
So would suggest keeping this in `lib/networks` and reading from there. 
Something like `isNativeCurrencyBridgingEnabled` ?